.github/workflows/ci.yml

```diff
@@ -73,9 +73,9 @@ jobs:
         uses: actions/cache@v2
         with:
           path: /tmp/.binfmt-cache
-          key: ${{ runner.os }}-binfmt-${{ github.sha }}
+          key: ${{ runner.os }}-binfmt-${{ matrix.target }}-${{ github.sha }}
           restore-keys: |
-            ${{ runner.os }}-binfmt-
+            ${{ runner.os }}-binfmt-${{ matrix.target }}-
```

Comment:
should this have a second restore key with just `${{ runner.os }}-binfmt-` ?

Reply:
Yes we could add that